linker/Mono.Linker/Utilities.cs

```diff
@@ -0,0 +1,227 @@
+﻿using Mono.Cecil;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Mono.Linker
+{
+	static class Utilities
+	{
+		internal static TypeReference GetBaseType (this TypeReference type)
```

Comment:
It'd be better to make methods intended for usage `public` and the rest `private`